internal/resolvers/twitch/check.go

```diff
@@ -19,17 +19,18 @@ func check(url *url.URL) bool {
 		return false
 	}
 
+	match, domain := resolver.MatchesHosts(url, domains)
+	if !match {
+		return false
+	}
+
 	// Find clips that look like https://clips.twitch.tv/SlugHere
-	if utils.IsDomain(url, "clips.twitch.tv") {
+	if domain == "clips.twitch.tv" {
```

Comment:
Since `resolver.MatchesHosts` already transforms our url to lower case we can be sure that it will be lowercased. With this logic I decided it's not worth adding `util` package and using `url.IsDomain`.